### PR TITLE
Updated the docs before release

### PR DESCRIPTION
**Pull request by muffato**
I updated the version number in `nextflow.config` and wrote the CHANGELOG (incl. #82)

@priyanka-surana : please find a release name !

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/sanger-tol/readmapping/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,35 @@
...
+- Updated the CI procedure to use "sanger-tol" rather than "nf-core" names.
+- [crumble](https://github.com/jkbonfield/crumble) now used to compress the
+  PacBio HiFi alignments.
+- Execution statistics now under `pipeline_info/readmapping/` (to be consistent
+  with the other sanger-tol pipelines).
+- All resource requirements (memory, time, CPUs) now fit the actual usage. This
+  is achieved by automatically adjusting to the size of the input whenever
+  possible.
+
+### Parameters
```
Comment (priyanka-surana) [inline CHANGELOG.md:20]: What about `use_work_dir_as_temp`?

**Review by priyanka-surana — APPROVED** [review]
Looks good. Added the release name: Norwegian Ridgeback
Do you want to describe the new parameter you added?

**Comment by gq1** [comment]
@priyanka-surana Can you remove the space in the diagram PPT file names?

File: CHANGELOG.md (hunk 1 above)
Comment (muffato) [reply CHANGELOG.md:20]: 🤦🏼‍♂️ Completely missed that one... Thank you !

**Pull request merged**